triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceFixture.scala

```diff
@@ -44,12 +46,44 @@ object TriggerServiceFixture {
       aesf: ExecutionSequencerFactory,
       ec: ExecutionContext): Future[A] = {
 
+    // Launch a toxiproxy instance . Wait on it to be ready to accept
+    // connections. Note that it binds to a hard-coded port (8474) here.
```

Comment:
Is the port configurable?

Reply:
Maybe. I have to look into it.

Reply:
Reading into the code, it looks like the port is configurable. So you can set it to another literal. Support for dynamic port allocation looks lacking. Overall I would say this test is likely to remain necessarily exclusive.

Reply:
In the Haskell test libraries we have a utility to find and return a free port, which one pass to services to avoid hardcoding the port number. Not suggesting you have to do it in this PR, but there might be something similar in the Scala test utils for doing that and avoiding exclusivity.

Reply:
That's interesting!